src/errors.rs

```diff
@@ -72,13 +72,13 @@ pub enum MentatError {
     InvalidVocabularyVersion,
 
     #[fail(display = "vocabulary {}/{} already has attribute {}, and the requested definition differs", _0, _1, _2)]
-    ConflictingAttributeDefinitions(String, ::vocabulary::Version, String, Attribute, Attribute),
```

Comment:
In all of these messages, ensure that all of these `u32` values are printed with "version {}" so that it's clear what the integer means.  So
```     #[fail(display = "vocabulary {}/version {} already has attribute {}, and the requested definition differs", _0, _1, _2)] ```